samples/drivers/led_gpio/src/main.c

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2018 Savoir-Faire Linux.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <device.h>
+#include <errno.h>
+#include <drivers/led.h>
+#include <sys/util.h>
+#include <sys/printk.h>
+#include <zephyr.h>
+
+#define LED_DEV_NAME DT_GPIO_LEDS_LEDS_DRIVER
+#define NUM_LEDS 4
+#define BLINK_DELAY_ON 500
+#define BLINK_DELAY_OFF 500
+#define DELAY_TIME K_MSEC(1000)
+
+static u8_t led_id[NUM_LEDS] = {DT_GPIO_LEDS_LED0_ID,
+				DT_GPIO_LEDS_LED1_ID,
+				DT_GPIO_LEDS_LED2_ID,
+				DT_GPIO_LEDS_LED3_ID};
+
+void main(void)
+{
+	struct device *led_dev;
+	int i, ret;
+
+	led_dev = device_get_binding(LED_DEV_NAME);
+	if (led_dev) {
+		printk("Found LED device %s\n", LED_DEV_NAME);
```

Comment:
Please use logging subsystem.

Reply:
Mostly the applications use the `printk` instead of logging subsystem.  I thought in app we should use the `printk`. I am not sure if it need to be changed